### PR TITLE
Universal Profiling: update native symbol upload instructions

### PR DESCRIPTION
**Pull request by athre0z**
We made various progress with symbolization in 8.8 and this PR updates the documentation accordingly. In general, things have become quite a bit more convenient for the user, so I ended up being able to delete a lot of things.

Fixes https://github.com/elastic/prodfiler/issues/3281

### CODE REVIEW

File: docs/en/observability/profiling-add-symbols.asciidoc [hunk 1]
```diff
@@ -4,148 +4,72 @@
...
-
-* No virtual frames for inline functions
-* No symbols for leaf frames (top-most frame in a trace)
-* No automatic insertion of debug symbols for OS packages
+Elastic hosts a public service with symbol information for all packages in the repositories
+of various popular linux distributions. Applications installed from these repositories will
+be automatically symbolized without any action required on the user's side.
 
-We hope to improve these limitations and remove them in later versions.
+The supported Linux distributions currently include:
```
Comment (mdbirnstiehl) [inline docs/en/observability/profiling-add-symbols.asciidoc:69]:
```suggestion
Elastic supports the following Linux distributions:

```

**Review by mdbirnstiehl — COMMENTED** [review]
I added a couple of comments. Let me know if you have any questions or comments.

**Review by florianl — APPROVED** [review]
nothing to add to @mdbirnstiehl feedback.

**Comment by athre0z** [comment]
Thanks for the feedback -- I applied all the suggested changes!

**Review by mdbirnstiehl — APPROVED** [review]
LGTM!

**Pull request merged**